config.yaml.defaults

```diff
@@ -97,10 +97,11 @@ log:
     # log only those that trigger a warning / error
     api_calls: False
 
-    # if true, log postgres database queries
+    # if true, enable SQL echoing.
     database: False
 
-    # set the debugging level (debug or info)
+    # if true, enable connection pool logging
+    # can also set the debugging level (debug or info)
```

Comment:
So can this flag be anything in `True, False, "debug", "info"`?

Reply:
I checked and True, False, and "debug" work. "info" did not for me so I took that off.